spec/lib/shanty/cli_spec.rb

```diff
@@ -112,6 +114,30 @@ module Shanty
 
         subject.run
       end
+
+      it('fails to run a command with config options if config is in incorrect format') do
+        expect(subject).to receive(:abort).with('Invalid config format "nic" should be [plugin]:[key] [value]')
```

Comment:
When you change the above to use `i18n`, you'll want to assert that it matches the output of `i18n.t` here rather than hardcoding the message here a second time.